Candidates.txt

```diff
@@ -1,2 +1,3 @@
 Bogdan Djuric bogdan-djuric
 Aleksandra your name goes here - replace second line with your name
+Ivana Jankovic ivana-jankovic
```

Comment:
When you are doing merge with some branch there is one very important step, to pull latest code from remote. So you should pull latest code from master and than do merge in your branch.